crud.py

```diff
@@ -0,0 +1,71 @@
+from sqlalchemy.orm import Session
+
+from db import models
+import schemas
+
+
+def get_book_list(db: Session, author_id: str | None = None):
+    queryset = db.query(models.DBBook)
```

Comment:
he function should return a list of books. You need to include a type hint and use .all() to fetch the results from the query.